plugins/module_utils/known_hosts.py

```diff
@@ -103,13 +103,11 @@ def not_in_host_file(self, host):
             continue
 
         try:
-            host_fh = open(hf)
+            with open(hf) as host_fh:
+                data = host_fh.read()
         except IOError:
             hfiles_not_found += 1
             continue
-        else:
-            data = host_fh.read()
-            host_fh.close()
```

Comment:
This is now changing behavior, since before only errors during opening resulted in `hfiles_not_found += 1` and `continue`, while now *all* errors during reading result in that action.

I think that change is OK, though. (Also `hfiles_not_found` is not used at all...)

Reply:
yeah, I'll make another PR later to remove it